internal/cli/cmd/playground/util.go

```diff
@@ -60,6 +60,20 @@ func cloudProviderRepoDir() (string, error) {
 	return filepath.Join(dir, cpDir), err
 }
 
+// cloudProviderRepoDirFromVersion cloud provider repo directory from kbcli version
+func cloudProviderRepoDirFromVersion(version string) (string, error) {
```

Comment:
I prefer to reuse the `cloudProviderRepoDir` function, add an arg `version`, if version is not empty, use it, otherwise use the `version.Version`